src/app/pages/common/entity/entity-job/entity-job.interface.ts

```diff
@@ -6,5 +6,23 @@ export enum EntityJobState {
   Failed = 'FAILED',
   Aborted = 'ABORTED',
   Success = 'SUCCESS',
-  Finished = 'FINISHED'
+  Finished = 'FINISHED',
+}
+
+export interface EntityJob {
+  abortable: boolean;
+  arguments: (number | string)[];
+  description: string | null;
+  error: string | null;
+  exc_info: { type: string; extra: string | null } | null;
+  exception: string | null;
+  id: number;
+  logs_excerpt: string | null;
+  logs_path: string | null;
```

Comment:
`string` and `number` can be `null` by default, you don't have to specify explicitly